dist/index.html

```diff
@@ -30,41 +36,47 @@ <h1 class="lg-heading">
               hesitate to contact me.
             </p>
           
-              <h2 href="#" class="connect">LET'S CONNECT</h2>
-                  <ul class="social-media">
-                    <li>
-                      <a href="#" class="icons"
-                        ><img src="./img/icons/twitter.svg"
-                      /></a>
-                    </li>
-                    <li>
-                      <a href="#" class="icons"
-                        ><img src="./img/icons/Linkedin.png"
-                      /></a>
-                    </li>
-                    <li>
-                      <a href="#" class="icons"
-                        ><img src="./img/icons/micro.svg"
-                      /></a>
-                    </li>
-                    <li>
-                      <a href="#" class="icons"><img src="./img/icons/git.svg" /></a>
-                    </li>
-                    <li>
-                      <a href="#" class="icons"><img src="./img/icons/hand.svg" /></a>
-                    </li>
-                  </ul>
-            </section>
+              <div>
+                <h2 href="#" class="connect">LET'S CONNECT</h2>
+              </div>
+                  <div>
+                    <ul class="social-media">
+                      <li>
+                        <a href="#" class="icons"
+                          ><img src="./img/icons/twitter.svg"
+                        /></a>
+                      </li>
+                      <li>
+                        <a href="#" class="icons"
+                          ><img src="./img/icons/Linkedin.png"
+                        /></a>
+                      </li>
+                      <li>
+                        <a href="#" class="icons"
+                          ><img src="./img/icons/micro.svg"
+                        /></a>
+                      </li>
+                      <li>
+                        <a href="#" class="icons"><img src="./img/icons/git.svg" /></a>
+                      </li>
+                      <li>
+                        <a href="#" class="icons"><img src="./img/icons/hand.svg" /></a>
+                      </li>
+                    </ul>
+                  </div>
+                </div>
+                </div>
+          </section>
 
 
-            <section class="works-card">
-                <div class="grid-container card-1">
-                    <div class="card-container">
-                      <div>
-                        <img class="card-img" src="./img/projects/Snapshoot Portfolio_1.png" alt="photo of my 1st project" />
-                      </div>
-                         <div class="text-container">
-                           <h2 class="card-title">Tonic</h3>
+          <section class="works-card">
+              <div class="grid-container grid-container-desktop">
```

Comment:
- [ ] Great work implementing required changes from the previous reviewer. However, there're few things you still need to work on before you move on to the next milestone. But you're almost there.
When you look at the Figma design very closely, you would see that both the image part and the image info share the card with equally plus the space between them. But your design looks a bit different from Figma design. Kindly look into that and fix.

### Your Design
![your-project-card](https://user-images.githubusercontent.com/26597432/236508880-ecec5e15-1e81-4e07-8604-c397f368038c.png)

### Figma Design
![temp-project-card](https://user-images.githubusercontent.com/26597432/236508897-fe2f65cf-c78a-41bf-a7e4-3cff00cbfdee.png)

Reply:
You did really well working on the changes by the previous reviewer, you're really close to moving to the next project but you have just some few changes 😄 

You've already adjusted the margin between the elements, how about you reduce the padding between them as it is on the high side at the moment to enable us to have a perfect replica of the design and please also remeber that `Back End Dev` is on one line and the font size used for that section is a bit bigger than what you have 

your design:

![image](https://user-images.githubusercontent.com/20537712/236566834-8f989ecc-6023-4092-8969-815b3b8b137b.png)

Figma design :

![image](https://user-images.githubusercontent.com/20537712/236566937-e7b79e6b-a9d8-4747-ba80-8b185638a48f.png)